src/Test/Utilities/Portable/FX/PinnedMetadata.cs

```diff
@@ -4,6 +4,7 @@
 using System.Collections.Immutable;
 using System.Reflection.Metadata;
 using System.Runtime.InteropServices;
+using Microsoft.CodeAnalysis;
```

Comment:
Consider reverting.

Reply:
I moved the helper method, so this is necessary to keep the file compiling.